README.md

```diff
@@ -15,3 +15,13 @@ For more specific documentation, visit one of the pages below:
 [Command-Line Quick Reference]: ./docs/command-line-quick-reference.md
 [Component Design System]: ./docs/component-design-system.md
 [How To Deploy]: ./docs/how-to-deploy.md
+
+## Stanging website
+
+We're currently in the process of developing a redesign of the sheltertech.org
+website! Until we are done and ready to flip the switch to the new design, we
+will be placing the new site pages under the `/new` URL path so that they can be
+easily previewed before we roll them out to the main site page routing.
+
+To view the current state of the new design on the staging website, please visit
+https://staging.sheltertech.org/new.
```

Comment:
Note that this URL won't work until this PR is merged into `main`, but once it does, it'll automatically be deployed to staging.